recipes/omniscidb-cuda/recipe/build.sh

```diff
@@ -0,0 +1,123 @@
+# add):
+export CXXFLAGS="`echo $CXXFLAGS | sed 's/-fPIC//'`"
+
+# Fixes https://github.com/Quansight/pearu-sandbox/issues/7
+#       https://github.com/omnisci/omniscidb/issues/374
+export CXXFLAGS="$CXXFLAGS -Dsecure_getenv=getenv"
+
+# Adjust OPENSSL_ROOT for conda environment. This ensures that
+# openssl is picked up from host environment:
+sed -i 's!/usr/local/opt/openssl!\'$PREFIX'!g' CMakeLists.txt
+
+# Avoid picking up boost/regexp header files from local system if
+# there:
+sed -i 's!/usr/local!\'$PREFIX'!g' CMakeLists.txt
+
+# Make sure that llvm-config and clang++ are from host environment,
+# otherwise UdfTest will fail:
+export PATH=$PREFIX/bin:$PATH
+
+# Fixes nvcc linker failure (nvcc assumes libraries are in
+# $PREFIX/lib64). Reconsider when new cudatoolk-dev is released:
+export LIBRARIES="$LIBRARIES -L$PREFIX/lib"
+
+# All these must be picked up from $PREFIX/bin
+CMAKE_ENV="CC=clang CXX=clang++"
+
+# Resolves `It appears that you have Arrow < 0.10.0`:
+export CFLAGS="$CFLAGS -pthread"
+export LDFLAGS="$LDFLAGS -pthread -lrt -lresolv"
+
+# fixes `undefined reference to
+# `boost::system::detail::system_category_instance'`:
+export CXXFLAGS="$CXXFLAGS -DBOOST_ERROR_CODE_HEADER_ONLY"
+
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++"
+
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DCMAKE_LIBRARY_PATH=$CUDA_HOME/lib64/stubs"
+export EXTRA_CMAKE_OPTIONS="$EXTRA_CMAKE_OPTIONS -DCUDA_TOOLKIT_ROOT_DIR=$CUDA_HOME"
+
+mkdir -p build
+cd build
+
+$CMAKE_ENV cmake -Wno-dev \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DCMAKE_BUILD_TYPE=release \
+    -DMAPD_DOCS_DOWNLOAD=off \
+    -DENABLE_AWS_S3=off \
+    -DENABLE_CUDA=on \
+    -DENABLE_FOLLY=off \
+    -DENABLE_JAVA_REMOTE_DEBUG=off \
+    -DENABLE_PROFILER=off \
+    -DENABLE_TESTS=on  \
+    -DPREFER_STATIC_LIBS=off \
+    $EXTRA_CMAKE_OPTIONS \
+    ..
+
+(which lscpu && which grep && which awk && which bc)
+if [ $? -eq 0 ]; then
+    export CORES_PER_SOCKET=`lscpu | grep 'Core(s) per socket' | awk '{print $NF}'`
+    export NUMBER_OF_SOCKETS=`lscpu | grep 'Socket(s)' | awk '{print $NF}'`
+    export NCORES=`echo "$CORES_PER_SOCKET * $NUMBER_OF_SOCKETS" | bc`
+    make -j $NCORES
+else
+    make -j $CPU_COUNT
+fi
+
+make install
+
+export LD_LIBRARY_PATH=${CUDA_HOME}/lib64:${LD_LIBRARY_PATH}
+
+# DEBUG:
+echo "CUDA_HOME=$CUDA_HOME"
+echo "PATH=$PATH"
+echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH"
+echo "LDFLAGS=$LDFLAGS"
+${CC} --print-sysroot
+ls -la `${CC} --print-sysroot`/lib/libcuda*
+
+# skip tests when libcuda.so is not available
+if [ "`ldd bin/initdb | grep "not found" | tr -d '[:space:]'`" == "libcuda.so.1=>notfound" ]; then
+    echo "SKIP RUNNING SANITY TESTS: libcuda.so.1 not found"
+else
+    # Add g++ include paths for astparser. This is used by the
+    # loadtime UDF support, resolves not found include files errors:
+    # cstdint, bits/c++config.h, features.h
```

Comment:
This happens when somebody calls `g++` with `-I/usr/include`. Do you know which program that does that?

Reply:
@pearu, ping

Reply:
Sorry, I did not miss your question but wanted to recheck if the comment is still valid.
Since it may be time-consuming to track this down  and the `else` block is not needed for azure anyway, I suggest postponing the answer to this question in feedstock (unless I'll figure this out earlier).

Reply:
ok, setting CPLUS_INCLUDE_PATH is unnecessary, after removing it all sanity tests pass ok